libs/core/src/lib/utils/functions/is-key.ts

```diff
@@ -0,0 +1,27 @@
+const keyMap: Map<string, { aliases: string[], keyCode: number }> = new Map(
+    [
+        ['ArrowRight', {aliases: ['ArrowRight', 'Right'], keyCode: 39}],
+        ['ArrowDown', {aliases: ['ArrowDown', 'Down'], keyCode: 40}],
+        ['ArrowLeft', {aliases: ['ArrowLeft', 'Left'], keyCode: 37}],
+        ['ArrowUp', {aliases: ['ArrowUp', 'Up'], keyCode: 38}],
+        [' ', {aliases: ['Space', 'Spacebar', ' '], keyCode: 32}],
+        ['Escape', {aliases: ['Escape', 'Esc'], keyCode: 27}],
+        ['Delete', {aliases: ['Delete', 'Del'], keyCode: 46}],
+        ['Enter', {aliases: ['Enter'], keyCode: 13}],
+        ['Tab', {aliases: ['Tab'], keyCode: 9}],
```

Comment:
Not being able to check for shift+tab could be an issue in the future

Reply:
For `shift/crtl/alt` + any key, there is flag in object keybaord event object
![image](https://user-images.githubusercontent.com/26483208/80200411-e491ec80-8622-11ea-9875-229bd0a2d9fa.png)